inc/devicesimcard.class.php

```diff
@@ -42,6 +42,16 @@ static function getTypeName($nb = 0) {
       return _n('Simcard', 'Simcards', $nb);
    }
 
+   public function defineTabs($options = []) {
+      $tabs = [];
+      $this->addImpactTab($tabs, $options);
+
+      // Merge with impact at second position
+      $tabs = $this->insertTabs(parent::defineTabs(), $tabs, 1);
```

Comment:
I really do not like. I guess you could add that tab in parent's `DefineTab` directly, based on configuration and current itemtype.
And this would make it de facto available for all concerned sub-types that may be added in the future.

Reply:
Done in ac7fe77.